### ISSUE TITLE
[Feature Request]: style template change

### ISSUE THREAD

**Issue by Tony931023** [azerothcore/wiki#754]
### Is your feature request related to a problem? Please describe.

it would be nice to be able to use another style for the wiki template, something more modern, without the left menu, the menu could be at the top like https://www.azerothcore.org/, improve the language selector since it doesn't stays in the selected language, add social links, such as discord, Facebook...

Some more ideas, drop-down list, a drop-down menu, or a pull-down menu

@pangolp @Helias

**Comment by pangolp** [comment]
Yes, totally, in fact, at the time, I had made some modifications, in the BlizzCMS wiki, which is a fork of this wiki. However, the style issue is not up to me, I could present an idea, or it would be good that several of us present one. But several changes could be made, no doubt. https://wow-cms.github.io/wiki/home. Although in reality, it has many of the styles, which the wiki has today. But nevertheless, you can change them, of course. The issue is that all parties agree, because it would be necessary to restructure the site, and I don't know if everyone can agree with that.

**Comment by pangolp** [comment]
It would be good to have designs so that we can then start a vote or something, and see which one we choose. I don't have the patience to design, but I love everything that is programming. We just have to have some design, either in figma, or even in images, that has the sizes and stuff, of each div, and the rest, no problem, I'll start programming it.

**Comment by Kitzunu** [comment]
What is the issue, apart from the shit language selector, with the wiki? What is wrong with having a bar at the left? Why are annoying dropdown menus preferred?

Now while I do agree that the current wiki is ass, not because of looks, but because of functionality and the general limitations of Markdown.

My suggestion would be to leave Github as the host and look into https://js.wiki/ (https://github.com/requarks/wiki)
This comes with great benefits: It looks great, it works better than markdown.
Only downside is that it needs to be hosted somewhere, which costs, and suggested by them is DigitalOcean Droplets or locally
(I know tc use this, which is not the reason why I am suggesting it, I just like it)

**Comment by pangolp** [comment]
It looks good, yes, it could be taken into account, but as long as we don't have a hosting, we won't be able to do it. If we had a hosting, we could also build it with Django and with many more alternatives, but for now, we have to work with what we have. Of course, if we have one, then it could be evaluated.

**Comment by pangolp** [comment]
I am working, to be able to set up a virtual environment, although I encountered some problems. In order to be able to start playing with the design and distribute the elements in a different way. Although I do not say that it will be the definitive form, with which the wiki will be, at least, it will be an alternative, so that we can analyze. It would be good, that everyone who can, try to create a theme, and that we could vote later, to see with which one we stay.

**Comment by pangolp** [comment]
Check this website if you want
https://jamstackthemes.dev/ssg/jekyll/
See if you find a theme you like.

**Comment by Kitzunu** [comment]
Those are for websites/bios, not for wikis

**Comment by Kitzunu** [comment]
I still don't see the issue with what we have today.

It is simple, it is clean, it is workable

**Comment by pangolp** [comment]
But some things could be improved, at least a little. I'm not saying to make a complete restructuring, but maybe some things, we could position them in another place, to make them look better.

**Comment by pangolp** [comment]
I particularly like this Python library for creating documentation: https://www.sphinx-doc.org/en/master/ and there is one particular topic, which I have seen in several places: https://github.com/readthedocs/sphinx_rtd_theme. But it would already be switching from Ruby to Python and I don't think they think it's a good idea. However, I will continue to study this library.

This is the format that has the theme, surely you have seen it in more than one place.

![theme](https://user-images.githubusercontent.com/2810187/177427828-49c0f7d2-5263-44aa-994d-eadf4a30efd6.jpg)

I am going to see if I can apply it to some module that I have created, to see how it looks.

**Comment by Kitzunu** [comment]
I am going to close this as what we got is good enough.
If you want to open up for more possible templates, make them in the git wiki repo
